tests/integration/targets/filter_hashids/tasks/main.yml

```
@@ -0,0 +1,58 @@
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+- name: Test valid hashable inputs
+  assert:
+    that:
+      - "single_int | community.general.hashids_encode | community.general.hashids_decode | first == single_int"
+      - "int_list | community.general.hashids_encode | community.general.hashids_decode | list == int_list"
+      - "(1,2,3) | community.general.hashids_encode | community.general.hashids_decode == (1,2,3)"
+
+- name: Test valid parameters
+  assert:
+    that:
+      - "single_int | community.general.hashids_encode(salt='test') | community.general.hashids_decode(salt='test') | first == single_int"
+      - "single_int | community.general.hashids_encode(alphabet='1234567890abcdef') | community.general.hashids_decode(alphabet='1234567890abcdef') | first == single_int"
+      - "single_int | community.general.hashids_encode(min_length=20) | community.general.hashids_decode(min_length=20) | first == single_int"
+      - "single_int | community.general.hashids_encode(min_length=20) | length == 20"
+
+- name: Test valid unhashable inputs
+  assert:
+    that:
+      - "single_float | community.general.hashids_encode | community.general.hashids_decode == ()"
+      - "arbitrary_string | community.general.hashids_encode | community.general.hashids_decode == ()"
+
+- name: Register result of invalid salt
+  debug:
+    var: "invalid_input | community.general.hashids_encode(salt=10)"
+  register: invalid_salt_message
+  ignore_errors: True
```

```suggestion
  ignore_errors: true
```
We should always use `true`/`false` in YAML.